app/models/concerns/human_enum_name_concern.rb

```diff
@@ -5,21 +5,25 @@ module HumanEnumNameConcern
 
   included do
     def human_enum_name(attribute)
-      enum_name = attribute.to_s.pluralize
+      @enum_names ||= Hash.new({})
```

Comment:
```suggestion
      @enum_names ||= {}
```

I think in this case you don't need the default value.